EngineLayer/PsmTsv/PsmFromTsv.cs

```diff
@@ -85,7 +85,7 @@ public class PsmFromTsv
 
         public PsmFromTsv(string line, char[] split, Dictionary<string, int> parsedHeader)
         {
-            var spl = line.Split(split);
+            var spl = line.Split(split).Select(p => p.Trim('\"')).ToArray();
```

Comment:
So this is some nightmare level regex stuff, but you should be able to deal with the edge case of a field enclosed by quotes using a conditional regex: https://docs.microsoft.com/en-us/dotnet/standard/base-types/alternation-constructs-in-regular-expressions#conditional-matching-based-on-a-valid-captured-group. Might need to combine the regex with a conditional that is triggered by a pattern match.

Reply:
Good to know it's possible, but let's hope it never comes to that